modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/RunDbNSFP.pm

```diff
@@ -146,6 +148,7 @@ sub run {
         my $aaref = $data{'aaref'};
         my $mutated_triplet =  $new_triplets->{$triplet_seq}->{$nucleotide_position}->{$alt};
         my $mutated_aa = $codonTable->translate($mutated_triplet);
+        next if ($aaalt ne $mutated_aa);
```

Comment:
Confusing variable `$aaalt` (but this is an old variable) might miss to add `a`s. It would makes sense to name it `$alt_aa` like we have `$mutated_aa`.